mycroft/skills/mycroft_skill/mycroft_skill.py

```diff
@@ -1072,7 +1088,8 @@ def translate_template(self, template_name, data=None):
         """Deprecated method for translating a template file"""
         return self._resources.load_template_file(template_name, data)
 
-    def add_event(self, name, handler, handler_info=None, once=False):
+    def add_event(self, name, handler, handler_info=None, once=False,
+                  speak_errors=True):
```

Comment:
this will need a companion change in ovos_workshop for OvosSkill class 

it overrides this in order to add killable_event support to mycroft-core, should probably add native support into ovos and make OvosSkill only patch mycroft-core